tests/src/main/java/com/hazelcast/stabilizer/tests/webContainer/TomcatServer.java

```diff
@@ -0,0 +1,71 @@
+package com.hazelcast.stabilizer.tests.webContainer;
+
+import org.apache.catalina.Context;
+import org.apache.catalina.Globals;
+import org.apache.catalina.LifecycleException;
+import org.apache.catalina.startup.ContextConfig;
+import org.apache.catalina.startup.Tomcat;
+
+import java.io.File;
+import java.util.Random;
+
+public class TomcatServer implements ServletContainer {
```

Comment:
Name is confusing. 

The actual implementation of the ServletContainer you call TomcatServer. And the actual test you call TomCatContainer.

Rename the test to 

TomcatContainerTest or TomcatTest

And rename the TomcatServer to TomcatServletContainer. This way it is more clear what is happening.